components/resources/rewards_strings.grdp

```diff
@@ -550,4 +550,140 @@
     It looks like this creator isn't set up to receive contributions via Web3 wallets yet.
   </message>
 
+  <!-- Connect UI -->
+  <message name="IDS_REWARDS_CONNECT_CUSTODIAL_TITLE" desc="">
+    Custodial accounts (requires ID verification)
+  </message>
+  <message name="IDS_REWARDS_CONNECT_CUSTODIAL_TOOLTIP" desc="">
+    Custodial accounts are accounts with trusted third-party services that hold your BAT for you.
+  </message>
+  <message name="IDS_REWARDS_CONNECT_SELF_CUSTODY_TITLE" desc="">
+    Self-custody
+  </message>
+  <message name="IDS_REWARDS_CONNECT_SELF_CUSTODY_TOOLTIP" desc="">
+    Self-custody gives you exclusive control over your BAT, along with complete responsibility for its security.
+  </message>
+  <message name="IDS_REWARDS_CONNECT_SELF_CUSTODY_NOTE" desc="">
+    By choosing a self-custody wallet, Rewards will be paid out as SPL-BAT on the Solana network.
```

Comment:
I know these are just moved, but this sounds a bit odd. Maybe `When a self-custody wallet is chosen,...`

Reply:
Let's use:
>With self-custody, your BAT earnings will be paid out as SPL-BAT on the Solana network.